amixr/manifest.json

```
@@ -0,0 +1,32 @@
+{
+  "display_name": "Amixr",
+  "maintainer": "ildar@amixr.io",
+  "manifest_version": "1.0.0",
+  "name": "amixr",
+  "metric_prefix": "amixr.",
+  "metric_to_check": "",
+  "creates_events": false,
+  "short_description": "Developer-friendly Alert Management with brilliant Slack integration",
+  "guid": "56e3b585-27b4-4a04-b528-9c1adfecf387",
+  "support": "contrib",
+  "supported_os": [
+    "linux",
+    "mac_os",
+    "windows"
+  ],
+  "public_title": "Datadog-Amixr Integration",
+  "categories": [
+    "monitoring",
+    "collaboration",
+    "notification"
+  ],
+  "type": "check",
+  "is_public": false,
```

```suggestion
  "is_public": true,
```

Make this integration available on docs page as well